static/js/apps/explore/app.tsx

```diff
@@ -214,6 +216,13 @@ export function App(props: { isDemo: boolean }): JSX.Element {
       pageMetadata.pageConfig &&
       pageMetadata.pageConfig.categories
     ) {
+      // If the response is a single PLACE_OVERVIEW tile, redirect to the place explorer.
+      if (isPlaceOverviewOnly(pageMetadata)) {
+        console.log('Redirecting to place explorer');
+        let url = 'place/';
+        url += pageMetadata.pageConfig.metadata["placeDcid"][0];
+        window.open(url, "_self");
```

Comment:
Use `window.location.replace(url);` instead. `window.open` adds a new record to the browser history, so hitting the back button takes the user back to a page where they'll just get redirected again

Reply:
Done - for the record the back and forward buttons still seem to work with window.open :)

Reply:
If i started from localhost:8080/ and ran a query, the back button seemed to work, but if i started from localhost:8080/explore/ and ran a query, then i'd see the intermediate page with window.open 🤷

Reply:
ah okay I had misunderstood your comments about starting from explore/, so I didnt try that. it looks like it works now with this approach though!